okta/samlApplicationSettingsSignOn.go

```diff
@@ -22,26 +22,26 @@ type SamlApplicationSettingsSignOn struct {
 	AcsEndpoints              []*AcsEndpoint            `json:"acsEndpoints,omitempty"`
 	AllowMultipleAcsEndpoints *bool                     `json:"allowMultipleAcsEndpoints,omitempty"`
 	AssertionSigned           *bool                     `json:"assertionSigned,omitempty"`
-	AttributeStatements       []*SamlAttributeStatement `json:"attributeStatements,omitempty"`
+	AttributeStatements       []*SamlAttributeStatement `json:"attributeStatements"`
```

Comment:
Isn't this file generated?  Shouldn't this update be made in the templates or generator?